runtime/js-compute-runtime/builtins/fastly.cpp

```diff
@@ -26,6 +26,9 @@ JS::PersistentRooted<JSObject *> Fastly::env;
 JS::PersistentRooted<JSObject *> Fastly::baseURL;
 JS::PersistentRooted<JSString *> Fastly::defaultBackend;
 bool Fastly::allowDynamicBackends = false;
+host_api::BackendConfig Fastly::defaultDynamicBackendConfig = host_api::BackendConfig{
+    std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt,
+    std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt};
```

Comment:
All those fields will default-initialize with nullopt, so you can omit them.
```suggestion
host_api::BackendConfig Fastly::defaultDynamicBackendConfig;
```